### ISSUE TITLE
Error in importing garage.envs

### ISSUE THREAD

**Issue by zhanpenghe** [rlworkgroup/garage#134]
With a fresh installation of ubuntu and conda environment, I encounter this error when running examples: 
`ImportError: No OpenGL rendering backend could be imported.` It is threw by dm_control.

This might be fix by installing glew and it should be included in setup_linux.sh.

**Comment by ryanjulian** [comment]
this is related to a problem in dm_control.

Workaround for Ubuntu 16.04
```sh
# Build GLFW because the Ubuntu 16.04 version is too old
# See https://github.com/glfw/glfw/issues/1004
RUN apt-get purge -y -v libglfw*
RUN git clone https://github.com/glfw/glfw.git && \
  cd glfw && \
  git checkout 0be4f3f75aebd9d24583ee86590a38e741db0904 && \
  mkdir glfw-build && \
  cd glfw-build && \
  cmake -DBUILD_SHARED_LIBS=ON -DGLFW_BUILD_EXAMPLES=OFF -DGLFW_BUILD_TESTS=OFF -DGLFW_BUILD_DOCS=OFF .. && \
  make -j"$(nproc)" && \
  make install && \
  cd ../../ && \
  rm -rf glfw
```

**Comment by ryanjulian** [comment]
https://github.com/deepmind/dm_control/issues/3

**Comment by gonzaiva** [comment]
Fixed with #258
